Zappy_IA_Report/NN/Code/Tutorial2.py

```
@@ -0,0 +1,25 @@
+data = keras.datasets.fashion_mnist
+(train_images, train_labels), (test_images, test_labels) = data.load_data()
+
+class_names = ['T-shirts/top', 'Trouser', 'Pullover', 'Dress', 'Coat', 'Sandal', 'Shirt', 'Sneaker', 'Bag', 'Ankle boot']
+
+train_images = train_images/255.0
+test_images = test_images/255.0
+
+model = keras.Sequential([
+	keras.layers.Flatten(input_shape=(28,28)),
+	keras.layers.Dense(128, activation="relu"),
+	keras.layers.Dense(10, activation="softmax")
```

Ruff (0.11.9)

Trailing comma missing

Add trailing comma

(COM812)